### DISCUSSION TITLE
Default values for initial infections

### DISCUSSION THREAD

**Discussion by robjohnnoble** [neherlab/covid19_scenarios#322] [General]
For the UK, your default parameter values lead the model to underestimate actual reported deaths by 79% (215 versus 1,019 as of 29th March). For the US, the underestimate is 85% (334 versus 2,191 as of 29th March). Hence the default parameter values are clearly inaccurate.

The "initial suspected cases" parameter has a big influence but it's unclear how the default values are chosen. For the UK, increasing initial suspected cases from 4,143 (default value) to 25,000 gives a much better fit to the mortality data. One can get a similar fit by either increasing R0 from 2.7 to 4.9 or doubling the % of cases that are severe and doubling the % of severe cases that are critical, but these alternative scenarios seem less likely.

Therefore it appears that the default numbers of initial suspected cases for the UK and US might be much too low.

It would be helpful to clarify on the About page how you choose the default numbers of initial suspected cases.

**Comment by rneher** [comment]
We are currently redoing this by fitting the model (incl an underreporting factor) to the initial part of the trajectory.  This will fit both R0 and initial numbers. Hopefully, this will address this concern soon.

**Reply by robjohnnoble** [reply]
That sounds like a good solution (though I guess there's quite a lot of uncertainty surrounding the underreporting factor). I look forward to seeing the new version.
